Description: Use the new soundtouch.pc file.
 The pkg-config soundtouch-1.0.pc file was obsolete and it is no longer
 provided, rely on the new soundtouch.pc instead.
Author: Miguel Colon <debian.micove@gmail.com>
Forwarded: no
Last-Update: 2011-02-21

Index: ardour/SConstruct
===================================================================
--- ardour.orig/SConstruct	2011-10-04 00:45:37.007369394 +0200
+++ ardour/SConstruct	2011-10-04 00:46:29.749835171 +0200
@@ -1104,7 +1104,7 @@
 
 #    libraries['flowcanvas'] = LibraryInfo(LIBS='flowcanvas', LIBPATH='#/libs/flowcanvas', CPPPATH='#libs/flowcanvas')
     libraries['soundtouch'] = LibraryInfo()
-    libraries['soundtouch'].ParseConfig ('pkg-config --cflags --libs soundtouch-1.0')
+    libraries['soundtouch'].ParseConfig ('pkg-config --cflags --libs soundtouch')
     # Comment the previous line and uncomment this for old versions of Debian:
     #libraries['soundtouch'].ParseConfig ('pkg-config --cflags --libs libSoundTouch')
 
